circuit_knitting/cutting/cutqc/__init__.py

```diff
@@ -33,3 +35,11 @@
     "create_dd_bin",
     "reconstruct_dd_full_distribution",
 ]
+
+warn(
+    f"The package {__name__} is deprecated and will be removed no sooner than Circuit Knitting Toolbox 0.8.0. "
+    "There is a new circuit cutting implementation in circuit_knitting.cutting that implements similar and improved functionality, which will be maintained going forward."
+    " See docs/tutorials/04_automatic_cut_finding.ipynb for a tutorial on automated LO cut-finding.",
```

Comment:
I suspect that many people using cutqc won't know what LO stands for.
Also, I wonder if a tutorial URL would be even better.

```suggestion
    " See https://qiskit-extensions.github.io/circuit-knitting-toolbox/circuit_cutting/tutorials/04_automatic_cut_finding.ipynb for a tutorial on automated cut-finding.",
```